feature/policy_forwarding/otg_tests/mpls_gre_udp_scale_test/README.md

```diff
@@ -0,0 +1,114 @@
+# PF-1.12 MPLS in GRE and MPLS in UDP Scale Test
+
+Building on TE-18.1 and PF-1.2, add scaling parameters
+
+## Topology
+
+* 2 ports as the 'input port set'
+* 4 ports as "uplink facing"
+* VRF configurations
+  * input vlans are distributed evenly across the 'input port set'
+
+## Test setup
+
+TODO: Complete test environment setup steps
+
+inner_ipv6_dst_A = "2001:aa:bb::1/128"
+inner_ipv6_dst_B = "2001:aa:bb::2/128"
+inner_ipv6_default = "::/0"
+
+ipv4_inner_dst_A = "10.5.1.1/32"
+ipv4_inner_dst_B = "10.5.1.2/32"
+ipv4_inner_default = "0.0.0.0/0"
+
+outer_ipv6_src =      "2001:f:a:1::0"
+outer_ipv6_dst_A =    "2001:f:c:e::1"
+outer_ipv6_dst_B =    "2001:f:c:e::2"
+outer_ipv6_dst_def =  "2001:1:1:1::0"
+outer_dst_udp_port =  "5555"
+outer_dscp =          "26"
+outer_ip-ttl =        "64"
+
+## Procedure
+
+### PF-1.12.1 Scale tests for MPLS in GRE and UDP
```

Comment:
please add a section that specifies exactly what gribi and config to generate and push.  You can reference another README containing  canonical, JSON format OC config for this if there is one that is appropriate.    I think your intent is to reference the gRIBI programming from TE-18.1 and the [JSON config from PF-1.14](https://github.com/openconfig/featureprofiles/blob/main/feature/policy_forwarding/otg_tests/mpls_gre_ipv4_encap_test/README.md?plain=1#L203)

is that right?